exchange/src/exchange/traders/trader_types/local_trader.hpp

```diff
@@ -0,0 +1,43 @@
+#pragma once
+
+#include "generic_trader.hpp"
+
+namespace nutc {
+namespace manager {
+
+class LocalTrader : public GenericTrader {
+public:
+    explicit LocalTrader(std::string algo_path, double capital) :
+        GenericTrader(std::move(algo_path), capital)
+    {}
+
+    constexpr TraderType
+    get_type() const override
+    {
+        return TraderType::LOCAL;
+    }
+
+    std::string
+    get_algo_id() const override
+    {
+        return get_id();
+    }
+
+    void
+    set_pid(pid_t pid) override
+    {
+        pid_ = pid;
+    }
+
+    pid_t
+    get_pid() const override
+    {
+        return pid_;
+    }
```

Comment:
use a reference

Reply:
pid_t is a signed integer, it shouldnt be returned as a reference

Reply:
No for the setter

Reply:
Setters should be done with references

Reply:
oh my bad i thought you meant getter